pkgs/tools/misc/cod/default.nix

```diff
@@ -0,0 +1,24 @@
+{ lib, fetchFromGitHub, buildGoPackage }:
+
+buildGoPackage rec {
```

Comment:
Why not `buildGoPackage`?

Reply:
it is this already. You probably did a typo.

Reply:
I'm sorry, I meant `buildGoModule`.